### PR TITLE
Add beautified maps

### LINKED ISSUE #20: Beautify SBGN
1. reduce crossing of edges, optimise positions, duplicate glyphs, if necessary
2. think about using SOME colors, after beautifying

provide files for both versions!

### PR DESCRIPTION
**Pull request by marisakaup**
Issue #20 

- [x] rearranged components
- [x] adapted label fonts and symbol sizes of the components
- [x] created one version in grey shades and one version highlighting CIS and SOCS3 components with colors

### CODE REVIEW

**Comment by marisakaup** [comment]
Suggest to delete yesterday´s colored map (model/sbgn/BachmannPD_tidyold_color.nwt). Not needed for archive.

**Comment by lknegendorf** [comment]
Hey @marisakaup thank you for your work! In my opinion, the map looks great now, but I have one suggestion: Would it be possible to color also the arcs showing the influence of SOCS3 and CIS in the corresponding colors? That would improve the understanding even more I think.

For the files in the pull request I would suggest that you just include the colored files and delete all other files (then we would have the "untidy" files with validated SBGN-ML and the "tidy" colored files.
Just to clarify: Deleting does not mean that the files are gone because the are stored in the Git version control system anyways, so we could reconstruct them if we need them even if you delete the branch.

Best, Leo

File: model/sbgn/BachmannPD_tidynew_color.nwt [hunk 1]
```diff
@@ -0,0 +1,1090 @@
...
+    </arc>
+    <arc id="nwtE_cb73aec2-24c0-4f18-a566-d537e8e42266" class="consumption" source="5d75f49f-297c-3ca0-3f1e-20780cd2babd" target="nwtN_042e2497-c05a-4c4d-9ef0-a60ef95ea14c">
+      <start x="-838.5220425910039" y="-2052.3364969389927"/>
+      <end x="-203.07948047782304" y="-1941.0290032656092"/>
+    </arc>
+    <arc id="nwtE_10fbe951-8acf-451a-a1dd-e29773c9a579" class="production" source="nwtN_042e2497-c05a-4c4d-9ef0-a60ef95ea14c" target="e08d1628-0d6a-e2e7-8532-ece8f502e322">
+      <start x="-203.07948047782304" y="-1937.6177608361597"/>
+      <end x="-835.4082023335773" y="-1884.0349365442985"/>
+    </arc>
+    <arc id="nwtE_3d4edd18-e61f-4506-ac4c-3f97b0b2a5b6" class="inhibition" source="nwtN_5096a7f8-9215-47f2-8250-00dd6a3813e5" target="nwtN_042e2497-c05a-4c4d-9ef0-a60ef95ea14c">
```
Comment (lknegendorf) [inline model/sbgn/BachmannPD_tidynew_color.nwt]: Maybe also the arcs from SOCS3 and CIS could be colored

File: model/sbgn/BachmannPD_tidynew_grey.nwt [hunk 2]
```diff
@@ -0,0 +1,1074 @@
+<?xml version='1.0' encoding='UTF-8' standalone='yes'?>
```
Comment (lknegendorf) [inline model/sbgn/BachmannPD_tidynew_grey.nwt]: Suggest to delete this file

**Review by lknegendorf — CHANGES_REQUESTED** [review]

File: model/sbgn/BachmannPD_tidynew_grey.nwt (hunk 2 above)
Comment (lknegendorf) [reply model/sbgn/BachmannPD_tidynew_grey.nwt]: See comment above

Comment (marisakaup) [reply model/sbgn/BachmannPD_tidynew_grey.nwt]: Done; commit fa75906

File: model/sbgn/BachmannPD_tidynew_color.nwt (hunk 1 above)
Comment (marisakaup) [reply model/sbgn/BachmannPD_tidynew_color.nwt]: Please review updated map c4da95a

**Pull request merged**